gridcapa-swe-commons/src/main/java/com/farao_community/farao/gridcapa_swe_commons/configuration/ProcessConfiguration.java

```diff
@@ -6,34 +6,44 @@
  */
 package com.farao_community.farao.gridcapa_swe_commons.configuration;
 
-import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
+import java.util.Map;
+
 /**
  * @author Marc Schwitzguébel {@literal <marc.schwitzguebel at rte-france.com>}
  */
 @Configuration
+@ConfigurationProperties(prefix = "swe-commons")
 public class ProcessConfiguration {
 
-    @Value("${swe-commons.zone-id}")
     private String zoneId;
-
-    @Value("${swe-commons.shift-max-iteration-number}")
     private Integer shiftMaxIterationNumber;
-
-    @Value("${swe-commons.modeling-authority-set}")
-    private String modelingAuthoritySet;
+    private Map<String, String> modelingAuthoritySet;
```

Comment:
I guess it might come from business jargon, but I find it quite original to store a modeling authority _set_ inside of a _map_. :)